examples/.gitignore

```diff
@@ -1,3 +1,4 @@
+monai-example/**
```

Comment:
This will work just fine. A nitpick-y minor thing to note is that it might be "better" to prefix these lines with `/` to indicate that the directory is specifically rooted parallel to this ignore file. Otherwise this will technically ignore any `monai-example` directory recursively underneath here (which there is likely *not* to be). E.g: 
```suggestion
/monai-example/**
```

Reply:
I assume this also applies to the `viame-example` directory?

Reply:
Yeah, I just missed clicking the multi-line comment thingy.